bucket/sophiapp.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "1.0.82",
+    "description": "An open-source app for configuring and fine-tuning Windows 10 & Windows 11.",
+    "homepage": "https://github.com/Sophia-Community/SophiApp/",
+    "license": "MIT",
+    "url": "https://github.com/Sophia-Community/SophiApp/releases/download/1.0.82/SophiApp.zip",
+    "hash": "194AB96708079BA6C74262EA39143FCDFC1437F52F511CE07813F59389271D31",
+    "extract_dir": "SophiApp",
+    "shortcuts": [
+        [
+            "SophiApp.exe",
+            "SophiApp"
+        ]
+    ],
+    "post_install": [
+        "if (Test-Path \"$dir\\SophiApp.exe.config.original\") {Remove-Item -Path \"$dir\\SophiApp.exe.config.original\" | Out-Null}"
+    ],
+    "persist": [
+        "SophiApp.exe.config",
+        "Logs"
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Sophia-Community/SophiApp/releases/download/$version/SophiApp.zip",
+        "hash": { 

```

```suggestion
        "hash": {
```
The trailing whitespace is causing CI to fail